regnety/dataset/imagenet.py

```diff
@@ -198,6 +208,7 @@ def random_crop(self, image: tf.Tensor, target: tf.Tensor) -> tuple:
         """
 
         cropped = tf.image.random_crop(image, size = (self.batch_size, 320, 320, 3))
+        cropped = tf.image.resize(cropped, size=(224,224))
         return cropped, target
     
     def center_crop_224(self, image: tf.Tensor, target: tf.Tensor) -> tuple:
```

Comment:
This name no longer makes sense, since the crop size may or may not be 224, depending on the config.